config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
This should not be here

Reply:
Or at least should be changed to `quay.io/opendatahub/airflow-operator` and the tag should be `openshift`

Reply:
Changed it